core/src/main/resources/META-INF/configurations/org.jahia.modules.modulemanager.configuration.constraints-default.yaml

```diff
@@ -0,0 +1,17 @@
+moduleLifeCycleConstraints:
```

Comment:
This will probably throw a lot of people off :)

Reply:
yeah should've also maybe renamed, but leaving as-is for now as name still makes sense :)